projects/js-packages/charts/package.json

```diff
@@ -21,11 +21,27 @@
 		"jest": "29.7.0",
 		"jest-environment-jsdom": "29.7.0",
 		"jest-extended": "4.0.2",
-		"typescript": "5.0.4"
+		"typescript": "5.0.4",
+		"@visx/axis": "^3.8.0",
+		"@visx/group": "^3.8.0",
+		"@visx/scale": "^3.8.0",
+		"@visx/shape": "^3.8.0",
+		"@types/react": "18.3.12",
+		"@types/react-dom": "18.3.1",
+		"react": "18.3.1",
```

Comment:
React seems already in peerDependencies. Plus if we depend on them, they should be in dependencies I believe.